package.json

```diff
@@ -11,6 +11,7 @@
     "@zeit/next-css": "^1.0.1",
     "@zeit/next-sass": "^1.0.1",
     "bootstrap": "^4.5.2",
+    "cookies": "^0.8.0",
```

Comment:
Why add package `cookies` if we already have `react-cookies`?

Reply:
It's not possible to use `react-cookies` in the api routes of nextjs (as far as I' know). `cookies` makes that possible.

Reply:
Ok. I suggest we look into just using one of these two packages. Given the complexity and timeliness of merging this PR, I'd rather leave it as is for now, and revise this in the future. Thank you.